samples/CognitiveServices.TextAnalytics/readme.md

```diff
@@ -5,6 +5,10 @@
 ``` yaml
```

Comment:
Let's not use a sample for it. I'd rather we had a swagger in `tests\TestsProjects` that we control and can write tests agains.